includes/register-posts.php

```diff
@@ -234,6 +242,62 @@ public function cpr_author_info_box($post)
                     <?php endforeach; ?>
                 </select>               
             </p>
+            <p>
+                <label for='author_title'>
+                    <?php echo __('Title', 'cpress') ?>:
+                </label>
+                <input type='text' name='author_title' id='author_title' value='<?= $author_title ?>'
+                         class="input-text regular-text" />
+            </p>
+            <p>
+                <label for='author_department'>
+                    <?php echo __('Department', 'cpress') ?>:
+                </label>
+                <input type='text' name='author_department' id='author_department' value='<?= $author_department ?>'
+                         class="input-text regular-text" />
+            </p>
+            <p>
+                <label for='author_location'>
+                    <?php echo __('Location', 'cpress') ?>:
+                </label>
+                <input type='text' name='author_location' id='author_location' value='<?= $author_location ?>'
+                         class="input-text regular-text" />
+            </p>
+            <p>
+                <label for='author_phone'>
+                    <?php echo __('Phone Number', 'cpress') ?>:
+                </label>
+                <input type='text' name='author_phone' id='author_phone' value='<?= $author_phone ?>'
+                         class="input-text regular-text" />
+            </p>
+            <p>
+                <label for='author_email'>
+                    <?php echo __('Email', 'cpress') ?>:
+                </label>
+                <input type='email' name='author_email' id='author_email' value='<?= $author_email ?>'
```

Comment:
Don't use shortcodes (<?= ?>), not all systems support them. Use <?php echo ; ?>.

Reply:
Hi @haydenyoung 
replaced it by full php tag, review it. If there is anything Please ping me on skype too.

Thanks